cfme/tests/cloud/test_keypairs.py

```diff
@@ -190,3 +186,25 @@ def test_download_private_key(keypair):
         initialEstimate: 1/4h
     """
     keypair.download_private_key()
+
+
+@pytest.mark.meta(automates=[1741635, 1747179])
+@test_requirements.multi_tenancy
+def test_tenantadmin_view_keypair(appliance, new_tenant_admin):
```

Comment:
Consider a test function name that better indicates that its an ownership test - `test_tenant_keypair_ownership`?

The assertion scope for the test case could be expanded, to do more assertion around the setting of ownership for a keypair (flash message asserts), as well as adding some parametrization to cover both a tenant that should _not_ have visibility and one that does.

Reply:
I have changed the test name to test_keypair_visibility_in_tenants since this is more of a keypair visibility test based on key pair ownership.

I will add parametrization to cover a tenant that doesn't have visibility in a follow up PR.